packages/muray/muray.cpp

```diff
@@ -31,12 +103,8 @@ void MuLabel(const v8::FunctionCallbackInfo<v8::Value> &args) {
 void MuInput(const v8::FunctionCallbackInfo<v8::Value> &args) {
     auto isolate = args.GetIsolate();
     auto context = isolate->GetCurrentContext();
-    int val = mu_textbox(&ctx, input_buf, sizeof(input_buf));
-    if (val & MU_RES_CHANGE)
-    {
-        mu_set_focus(&ctx, ctx.last_id);
-        args.GetReturnValue().Set(v8::String::NewFromUtf8(isolate, input_buf).ToLocalChecked());
-    }
+    auto id = *v8::String::Utf8Value(isolate, args[0]->ToString(context).ToLocalChecked());
+    inputs.render(&ctx, id, args);
```

Comment:
Getting the argument and putting it inside the `id` variable makes the example completely broken on my machine. I fixed it by putting `*v8::String::Utf8Value(isolate, args[0]->ToString(context).ToLocalChecked())` directly as an argument of the `inputs.render` function.

Reply:
I also just fixed it by doing this instead:

```suggestion
    auto id = v8::String::Utf8Value(isolate, args[0]->ToString(context).ToLocalChecked());
    inputs.render(&ctx, *id, args);
```